unity-renderer/Assets/Scripts/MainScripts/DCL/Components/LoadableShapes/NFTShape/NFTShapeLoaderController.cs

```diff
@@ -159,13 +167,16 @@ IEnumerator FetchNFTImage()
             },
             (error) =>
             {
-                Debug.LogError($"Couldn't fetch NFT: '{darURLRegistry}/{darURLAsset}' {error}");
+                Debug.LogError($"{COULD_NOT_FETCH_NFT_FROM_API.Replace("{darURLRegistry}", darURLRegistry).Replace("{darURLAsset}", darURLAsset)} {error}. {DOES_NOT_SUPPORT_POLYGON}");
```

Comment:
Could we change al those `Replace` for the errors? It doesn't make much sense creating a constant like `COULD_NOT_FETCH_NFT_FROM_API` while at the same time using a hardcoded string in the line like `"{darURLRegistry}"`. Using `Replace` here is like too much for something so little 🤣
Is probably better just to use `string.Format`
i.e:
```
COULD_NOT_FETCH_NFT_FROM_API = "Couldn't fetch NFT: '{0}/{1}'.";
 Debug.LogError(string.Format(COULD_NOT_FETCH_NFT_FROM_API, darURLRegistry, darURLAsset));
```

Reply:
You're totally right, I have changed all these `.Replace()` by `string.Format()`. Commited in https://github.com/decentraland/unity-renderer/pull/909/commits/662d0ab4ad8be0b33b488369659f9d643a6e651b. Thanks for the feedback, Pato!